content/spin/install.md

```diff
@@ -130,6 +133,8 @@ $ ./target/release/spin --help
 $ sudo apt-get install build-essential libssl-dev pkg-config
 ```
 
+This does not install any Spin templates or plugins. For a starter list, see the [Installing Templates and Plugins section](#installing-templates-and-plugins).
```

Comment:
Let's use the same wording for all of the "bare" install cases (this wording or the one above).

Reply:
This wording works. Thanks.